src/OneExplorer/OneExplorer.ts

```diff
@@ -565,7 +568,14 @@ class OneccRunner extends EventEmitter {
     });
   }
 
-  private onFinishedRunningOnecc() {
-    vscode.window.showInformationMessage(`Successfully completed`);
+  private onFinishedRunningOnecc(val: ExitCode|SelfKilled) {
+    const exitCodeSuccess = '0';
+    if (val === exitCodeSuccess) {
+      vscode.window.showInformationMessage(`Successfully completed.`);
+    } else if (val === 'SelfKilled') {
```

Comment:
Instead of `'0'`, `exitCodeSuccess` is used.
So how about using `SelfKilled` instead of `'SelfKilled'` for consistency? :)

```suggestion
    } else if (val === SelfKilled) {
```

Reply:
With ```if (val === SelfKilled)```, I got a compilation error. 😢 
```
SelfKilled only refers to a type, but is being used as a value here.ts(2693)
```

Reply:
I understand that @llFreetimell wants like

```ts
const selfKilled = 'SelfKilled';
// ...
{ else if (val === selfKilled) {
```

Reply:
I didn't notice that `SelfKilled` is not string but a literal type...

```ts
export type ExitCode = string;
export type SelfKilled = 'SelfKilled';
```

If so, I think `SelfKilled` is not used anywhere in both main branch and current PR... @.@

- Main branch
`SelfKilled` in the `resolve('SelfKilled')` may be handled by `string`, not `SelfKilled`.
https://github.com/Samsung/ONE-vscode/blob/746f1337a13f0c0187fb4be501107b960bca77f0/src/Project/ToolRunner.ts#L69
https://github.com/Samsung/ONE-vscode/blob/746f1337a13f0c0187fb4be501107b960bca77f0/src/Project/ToolRunner.ts#L42

- Current PR
`SelfKilled` in `val === 'SelfKilled'` maybe handled by `ExitCode`(`string`), not `Selfkilled`.
  ```ts
    private onFinishedRunningOnecc(val: ExitCode|SelfKilled) {
      const exitCodeSuccess = '0';
      if (val === exitCodeSuccess) {
        vscode.window.showInformationMessage(`Successfully completed.`);
      } else if (val === 'SelfKilled') {
  ```

Am I understanding correctly?

Reply:
Merging `SelfKilled` and `ExitCode` into `ExitCode` also works and I tried that way at the beginning. :-)

However, the reason I wanted to decided to handle this separately is because of the following reason:
https://github.com/Samsung/ONE-vscode/blob/d4f44b8eb382657b6a57574dbb3e6ad63df9d91e/src/Project/ToolRunner.ts#L56-L62

Please refer to https://nodejs.org/api/child_process.html#event-exit

I think that we'd better keep this separation and make the semantics of `ExitCode` just like mentioned in the above reference. 😸